server/lib/src/server/migrations.rs

```diff
@@ -894,6 +896,35 @@ impl<'a> QueryServerWriteTransaction<'a> {
                 err
             })?;
 
+        let mut modlist: Vec<_> = Vec::new();
```

Comment:
Can't you just do:

```
let filter = f_lt(Attribute::Uuid, PartialValue::Uuid( uuid::uuid!("00000000-0000-0000-0001-000000000000") ));
let modlist = m_pres(Attribute::Class, EntryClass::Object)

server.internal_modify(&filter, &modlist)
```

I think it may need a change in server/lib/src/valueset/uuid.rs to impl fn lessthan though .... (see server/lib/src/valueset/uint32.rs for inspiration )

Reply:
I guess I can, didn't see the need to add more internal functionality to do it... 😄 Added lessthan... I'll have at go at the filter at some point.

Reply:
Welp. That broke all the replication tests because it adds it to *everything*, including schema objects:

```
de763aa4-b78b-49d0-bff0-7e7f3b00ee98 TRACE    ┝━ 📍 [trace]: Entry::validate -> target | self.attrs: {"attributename": ValueSetIutf8 { set: {"name"} }, "class": ValueSetIutf8 { set: {"attributetype", "builtin", "object", "system"} }, "description": ValueSetUtf8 { set: {"The shortform name of an object"} }, "index": ValueSetIndex { set: [Equality, Presence] }, "last_modified_cid": ValueSetCid { set: [00000000000001709380154264643000-5919e721-9c78-48ed-ab74-f11d0efc5d19] }, "multivalue": ValueSetBool { set: [false] }, "phantom": ValueSetBool { set: [false] }, "replicated": ValueSetBool { set: [true] }, "sync_allowed": ValueSetBool { set: [true] }, "syntax": ValueSetSyntax { set: [Utf8StringIname] }, "unique": ValueSetBool { set: [true] }, "uuid": ValueSetUuid { set: [00000000-0000-0000-0000-ffff00000002] }}
de763aa4-b78b-49d0-bff0-7e7f3b00ee98 DEBUG    ┝━ 🐛 [debug]: invalid class: "builtin" | event_tag_id: 0
de763aa4-b78b-49d0-bff0-7e7f3b00ee98 ERROR    ┝━ 🚨 [error]: Schema Violation in refresh validate InvalidClass(["builtin"]) | event_tag_id: 1
de763aa4-b78b-49d0-bff0-7e7f3b00ee98 ERROR    ┕━ 🚨 [error]: Failed to refresh schema entries
thread 'repl::tests::qs_test_repl_refresh_basic' panicked at server/lib/src/repl/tests.rs:120:5:
assertion failed: repl_initialise(&mut server_b_txn,
            &mut server_a_txn).and_then(|_| server_a_txn.commit()).is_ok()
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
Fixed it, added some more filters. The "lessthan" feels icky, but I guess as long as there's only one UUID for an object it should be fine :P